docs/Intro.md

```diff
@@ -33,7 +33,7 @@ No matter where your game lives, it should have a first-class experience in Disc
 - Ask to Join and Spectate their friends' games
```

Comment:
If we are updating parts of the intro, spectating is no longer an available feature as described here.

Reply:
Hmm, I've not used the spectating functions myself so I'd need a source on that. Considering all of the spectating docs are still on the site.

Reply:
Refer to this by Mason in Discord Devs: https://discordapp.com/channels/613425648685547541/613425918748131338/743499318484402177

Reply:
would probably be better to make a separate PR removing all spectating docs/references

Reply:
I agree with you on that, so resolving this for now.